src/frontend/src/components/public/PIAFormTabs/storingPersonalInformation/index.tsx

```diff
@@ -340,30 +340,65 @@ const StoringPersonalInformation = () => {
       <section className="form__section">
         <h3 className="py-3">{Messages.PersonalInformation.H3Text.en}</h3>
         <div className="card-wrapper py-5 px-5">
-          <div>
-            <p>{Messages.PersonalInformation.StoredOutsideCanada.en}</p>
-            {piOutsideOfCanadaRadios.map((radio, index) => (
+          {!isReadOnly ? (
+            <p className="text__font-weight--700">
```

Comment:
should we use \<Strong\> instead of defining the class? \<Strong\> is understood by accessibility tools, class definition are not from my understanding. 
https://www.w3.org/WAI/WCAG21/Techniques/html/H49

Would love our team to sync up on this :). My 2 cents.